ansible_base/models/organization.py

```diff
@@ -0,0 +1,30 @@
+"""Organization models."""
+from django.conf import settings
+from django.db import models
+
+from .common import UniqueNamedCommonModel
+
+
+class AbstractOrganization(UniqueNamedCommonModel):
+    """An abstract base class for organizations."""
+
+    class Meta:
+        abstract = True
+
+    description = models.TextField(
+        null=False,
+        default="",
+        help_text="The organization description.",
+    )
+
+    users = models.ManyToManyField(
+        settings.AUTH_USER_MODEL,
+        related_name="organizations",
+        help_text="The list of users in this organization.",
+    )
+
+    teams = models.ManyToManyField(
```

Comment:
In Gateway / AWX this is a reverse ForeignKey. That is, the `Team.organization` field is just a normal ForeignKey.

Reply:
I confirmed it with @newswangerd and according to the information I got, this is supposed to be many to many relationship. Can we all synchronize on this?

Reply:
@AlanCoding I was working under the assumption that a team could live in two orgs, but if that's not the case in AWX, then I don't see the need to complicate it. @cutwater you can change this to a foreign key.